specificatie/reisdocument.yaml

```diff
@@ -6,90 +6,102 @@ info:
       properties:
-        reisdocumentnummer:
-          $ref: '#/components/schemas/Reisdocumentnummer' 
-        indicatieGeldig:
+        datum:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+        aanduiding:
           $ref: 'common.yaml#/components/schemas/Waardetabel'
+        inOnderzoek:
+          $ref: '#/components/schemas/InhoudingOfVermissingInOnderzoek'
+    InhoudingOfVermissingInOnderzoek:
+      type: object
+      properties:
+        datum:
+          type: boolean
+        aanduiding:
+          type: boolean
+        datumIngangOnderzoek:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    ReisdocumenthouderBasis:
+      type: object
+      properties:
         burgerservicenummer:
-          $ref: '#/components/schemas/Burgerservicenummer'
-    ReisdocumentUitgebreid:
+          $ref: 'persoon.yaml#/components/schemas/Burgerservicenummer'
+        geheimhoudingPersoonsgegevens:
+          $ref: 'persoon.yaml#/components/schemas/GeheimhoudingPersoonsgegevens'
+        opschortingBijhouding:
+          $ref: 'opschorting-bijhouding.yaml#/components/schemas/OpschortingBijhouding'
+    GbaReisdocumenthouder:
       allOf:
-        - $ref: '#/components/schemas/Reisdocument'
+        - $ref: '#/components/schemas/ReisdocumenthouderBasis'
         - type: object
-          description: |
-                        Een document dat nodig is voor reizen naar het buitenland.
-
-                        - indicatieGeldig geeft aan of het reisdocument op dit moment nog geldig en in het bezit van de persoon is. Wordt gevuld op basis van de waarden die voorkomen in de tabel 'Geldig_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.
-                        - soortReisdocument wordt gevuld op basis van de waarden die voorkomen in de tabel 'Nederlands_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.
           properties:
-            soortReisdocument:
-              $ref: 'common.yaml#/components/schemas/Waardetabel'
-            datumEindeGeldigheid:
-              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            inOnderzoek:
+              $ref: 'gba-inonderzoek.yaml#/components/schemas/GbaInOnderzoek'
+    Reisdocumenthouder:
+      allOf:
+        - $ref: '#/components/schemas/ReisdocumenthouderBasis'
+        - type: object
+          properties:
+            inOnderzoek:
+              $ref: '#/components/schemas/ReisdocumenthouderInOnderzoek'
+    ReisdocumenthouderInOnderzoek:
+      type: object
+      properties:
+        burgerservicenummer:
+          type: boolean
+        datumIngangOnderzoek:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    Reisdocumentnummer:
+      type: string
+      pattern: ^[0-9]{10}$
```

Comment:
Het reisdocumentnummer is alfanumeriek en 9 lang. pattern ^[0-9]{10}$ is dus te strikt (alleen cijfers) en niet strikt genoeg (10 ipv 9 lang). @fransrijkers @DannySiebelink kunnen we hier preciezer in zijn? Welke tekens kunnen worden gebruikt in een reisdocumentnummer?

Reply:
aangepast naar ^[A-Z0-9]{9}$

Reply:
ik vind voorbeelden met kleine letters, punt, komma, streepje (-), spatie, diakriet (bijv. ì) in het reisdocumentnummer. 
Ik vind ook reisdocumentnummers die minder dan 9 tekens lang zijn

Ik denk dat dit betekent dat tenminste voor de response meer dan alleen exact 9 hoofdletters of cijfers moet worden geaccepteerd. Maar wellicht ook in de parameter